Makefile

```diff
@@ -51,7 +51,14 @@ check: usig-build usig-check
 	go test -short -race ./...
 
 lint: usig-go-wrapper
-	gometalinter --vendor --enable="gofmt" --enable="misspell" --exclude='.+[.]pb[.]go' --deadline=5m ./...
+	golangci-lint run --skip-files "_test\.go" \
+		--disable-all \
+		-E govet -E typecheck -E deadcode -E gocyclo -E golint \
+		-E varcheck -E structcheck -E maligned -E errcheck \
+		-E megacheck -E dupl -E ineffassign -E interfacer \
+		-E unconvert -E goconst -E gosec \
+		-E gofmt -E misspell \
+		./...
```

Comment:
This command line becomes too busy. Let's use a [config file](https://github.com/golangci/golangci-lint/blob/v1.15.0/README.md#config-file) to specify the options.